src/components/common/DropDownMenu/dropDownMenu.tsx

```diff
@@ -0,0 +1,123 @@
+import { font } from "@/styles/font";
+import { useState } from "react";
+import { color } from "@/styles/color";
+import styled, { css } from "styled-components";
+import ArrowDropDownMenu from "../Icon/ArrowDropDownMenu";
+import { DropdownItemType } from "./type";
+
+interface PropsType {
+  desc: string;
+  dropdownMenuDatas: DropdownItemType[];
+  width: string;
+}
+
+const DropDownMenu = ({
+  desc,
+  dropdownMenuDatas,
+  width = "320px",
+}: PropsType) => {
+  const [isOpenDropdownMenu, setIsOpenDropdownMenu] = useState(false);
+  const [dropdownMenuText, setDropdownMenuText] =
+    useState("옵션을 선택해 주세요");
```

Comment:
```suggestion
  const [selectedItemText, setSelectedItemText] = useState("옵션을 선택해 주세요");
```

이 네이밍이 좀 더 적절할것같다는 생각이 듭니다 ~ 참고 부탁드립니다